packages/appsync-emulator-serverless/tester.js

```diff
@@ -41,10 +28,10 @@ const create = async ({ serverless, schemaPath, port = 0 } = {}) => {
     subscriptions,
   });
 
-  const close = () => {
+  const close = async () => {
     server.close();
     // schema deletes tables so we must close the emulator after.
-    return schemaClose();
+    await schemaClose();
```

Comment:
Is this a stylistic change, or something to make jest happy?

Reply:
stylistic (result of some reverted refactoring)